tests/test_ensemble.py

```diff
@@ -195,6 +195,10 @@ def test_emptyens():
     assert isinstance(emptydf, pd.DataFrame)
     assert emptydf.empty
 
+    emptyparams = ens.parameters
+    assert isinstancve(emptyparams, pd.DataFrame)
```

Comment:
```suggestion
    assert isinstance(emptyparams, pd.DataFrame)
```

Reply:
whops,  that was not an intentional part of the PR...